src/utils/nameModifiers.ts

```diff
@@ -12,9 +12,9 @@ export const FREE_FILE_SUFFIX = '__WC_FREE';
 export const CONTRACT_INFIX = '__WC__';
 
 // Used in IdentifierManglerPass
```

Comment:
Could you add `CairoStubProcessor` here as a possible location

Reply:
Ok.